### PR TITLE
Added code

### PR DESCRIPTION
**Pull request by aseemkthakur**
Please fill out the survey before submitting the pull request. Thanks!

🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

How many hours did you spend on this assignment? 
**1 hour**

Please fill in one error and/or error message you received while working on this assignment.

What part of the assignment did you spend the most time on?

Comfort Level (1-5):
3

Completeness Level (1-5):
3

What did you think of this deliverable?
It was quite fun project

Is there anything in this code that you feel pleased about?
being able to use a math trick

### CODE REVIEW

**Comment by kai-rocket** [comment]
@aseemkthakur do you think you could fill out the above survey for the assignment? thank you!

File: script.js [hunk 1]
```diff
@@ -1,4 +1,87 @@
...
+var player1Dice1;
+var player1Dice2;
+var player1Pick;
+var player2Dice1;
+var player2Dice2;
+var player2Pick;
+
+var diceRoll = function () {
+  // number from 0 to 1
+  var digit = Math.random();
```
Comment (kai-rocket) [inline script.js:11]: `digit` is misleading, `Math.random` returns a decimal

File: script.js [hunk 2]
```diff
@@ -1,4 +1,87 @@
...
+var player2turn = function () {
+  player2Dice1 = diceRoll();
+  player2Dice2 = diceRoll();
+  gameMode = 'player2picks';
+};
+
+var player1P = function (input) {
+  var result;
+  if (input == 1) {
+    result = (player1Dice1 * 10) + player1Dice2;
```
Comment (kai-rocket) [inline script.js:34]: Nice math!

File: script.js [hunk 3]
```diff
@@ -1,4 +1,87 @@
...
+  }
+  else if (input == 2) {
+    result = (player1Dice2 * 10) + player1Dice1;
+  }
+  gameMode = 'player2rolls';
+  return result;
+};
+
+var player2P = function (input) {
```
Comment (kai-rocket) [inline script.js:43]: `player1P` and `player2P` use the same logic, any way to refactor the logic so we don't repeat ourselves? Repeating logic increases possibility of bugs and makes code harder to maintain.

File: script.js [hunk 4]
```diff
@@ -1,4 +1,87 @@
...
+  gameMode = 'player1picks';
+};
+
+var player2turn = function () {
+  player2Dice1 = diceRoll();
+  player2Dice2 = diceRoll();
+  gameMode = 'player2picks';
+};
+
+var player1P = function (input) {
```
Comment (kai-rocket) [inline script.js:31]: Consider using more descriptive function names. Fn names should start with a verb for clarity

File: script.js [hunk 5]
```diff
@@ -1,4 +1,87 @@
...
+  gameMode = 'gameResult';
+  return result;
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var myOutputValue;
+  if (gameMode == 'player1rolls') {
+    player1turn();
```
Comment (kai-rocket) [inline script.js:58]: `player1Turn` for camelCase

File: script.js [hunk 6]
```diff
@@ -1,4 +1,87 @@
...
+    myOutputValue = 'Welcome Player1. <br> You rolled Dice 1:' + player1Dice1 + ' and Dice 2:' + player1Dice2 + '. <br> Choose the order of the dice.';
+    return myOutputValue;
```
Comment (kai-rocket) [inline script.js:60]: Great work returning in the conditional to reduce nesting in the function

File: script.js [hunk 7]
```diff
@@ -1,4 +1,87 @@
...
+    result = (player2Dice1 * 10) + player2Dice2;
+  }
+  else if (input == 2) {
+    result = (player2Dice2 * 10) + player2Dice1;
+  }
+  gameMode = 'gameResult';
+  return result;
+};
+
 var main = function (input) {
```
Comment (kai-rocket) [inline script.js:55]: Great work keeping the logic simple and functional!

File: script.js [hunk 8]
```diff
@@ -1,4 +1,87 @@
...
+    myOutputValue = 'Player1, you chose Dice ' + input + ' fist. <br> Your number is ' + player1Pick + '. <br> It is now Player 2\'s turn';
+    return myOutputValue;
+  }
+  if (gameMode == 'player2rolls') {
+    player2turn();
+    myOutputValue = 'Welcome Player2. <br> You rolled Dice 1:' + player2Dice1 + ' and Dice 2:' + player2Dice2 + '. <br> Player 1\'s score is ' + player1Pick + '<br> Choose the order of the dice.';
+    return myOutputValue;
+  }
+  if (gameMode == 'player2picks') {
+    player2Pick = player2P(input);
```
Comment (kai-rocket) [inline script.js:73]: Consider adding inline comments to help explain what your code does

**Review by kai-rocket — COMMENTED** [review]
Clean and clear! Consider abstracting repetitive logic and adding comments for clarity. Can work on variable naming, will be more important for more complex programs.

File: script.js (hunk 8 above)
Comment (aseemkthakur) [reply script.js:73]: Thanks for the feedback :) @kai-rocket

**Pull request closed without merging**